pkg/modules/quickfilter/core/store.go

```diff
@@ -0,0 +1,85 @@
+package core
+
+import (
+	"context"
+	"database/sql"
+	"github.com/SigNoz/signoz/pkg/sqlstore"
+	"github.com/SigNoz/signoz/pkg/types/quickfiltertypes"
+	"github.com/SigNoz/signoz/pkg/valuer"
+)
+
+type store struct {
+	store sqlstore.SQLStore
+}
+
+// NewStore creates a new SQLite store for quick filters
+func NewStore(db sqlstore.SQLStore) quickfiltertypes.QuickFilterStore {
+	return &store{store: db}
+}
+
+// GetQuickFilters retrieves all filters for an organization
+func (s *store) Get(ctx context.Context, orgID valuer.UUID) ([]*quickfiltertypes.StorableQuickFilter, error) {
+	filters := make([]*quickfiltertypes.StorableQuickFilter, 0)
+
+	err := s.store.
+		BunDB().
+		NewSelect().
+		Model(&filters).
+		Where("org_id = ?", orgID).
+		Order("signal ASC").
+		Scan(ctx)
+
+	if err != nil {
+		return filters, err
+	}
+
+	return filters, nil
+}
+
+// GetSignalFilters retrieves filters for a specific signal in an organization
+func (s *store) GetBySignal(ctx context.Context, orgID valuer.UUID, signal string) (*quickfiltertypes.StorableQuickFilter, error) {
+	filter := new(quickfiltertypes.StorableQuickFilter)
+
+	err := s.store.
+		BunDB().
+		NewSelect().
+		Model(filter).
+		Where("org_id = ?", orgID).
+		Where("signal = ?", signal).
+		Scan(ctx)
+
+	if err != nil {
+		if err == sql.ErrNoRows {
+			return nil, nil
```

Comment:
```suggestion
return nil, sqlstore.WrapNotFoundf(...)
```

In go, a caller expects an error to be present. If the error is not nil, the caller will expect the next argument to be not-nil.

If you want to do something specfically on `NotFound`, use the `sqlstore.WrapNotFoundf` function. Then on the caller side, use `errors.Ast(err, errors.TypeNotFound)` to do something meaningful.